third_party/ibis/ibis_oracle/__init__.py

```diff
@@ -55,7 +58,7 @@ def do_connect(
             sa_url = sa.engine.url.make_url(url)
 
         self.database_name = sa_url.database
-        engine = sa.create_engine(sa_url, poolclass=sa.pool.StaticPool)
+        engine = sa.create_engine(sa_url, poolclass=sa.pool.StaticPool, arraysize=self.arraysize)
```

Comment:
Can you run `black` on the code to satisfy the failing lint test please.

Reply:
linter checks completed. Refactored code included in latest PR